.circleci/config.yml

```diff
@@ -104,7 +104,7 @@ jobs:
                 ~/cmake-3.16.5-Linux-x86_64/bin/ctest --output-on-failure -timeout 120
   build_macos_release:
     macos:
-      xcode: "12.0.0"
+      xcode: "12.5.1"
```

Comment:
If i'm not mistaken, there will be a new version to come, see with Victor about that, for reference: https://github.com/LedgerHQ/lib-ledger-core-build-env/pull/9

Reply:
@jcoatelen-ledger
How that build env update can fix xcode version?

Reply:
it's not related, I made a confusion sorry